2.arrayOccurenceFilter.js

```diff
@@ -11,7 +11,17 @@
 // occurrenceFilter([1, 3, 4, 1, 9, 1, 3, 4, 3, 1, 2], 2) returns [1, 3, 4] 
 
 const occurrenceFilter = (arr, n) => {
-
+    let output = []
+    let count = 0
+    arr.forEach((el) => {
+        if(arr[el] === el){
+            count ++
+        }
+        if(count >= n){
+            output.arr[el]
+        }
+    })
+    return output
 }
```

Comment:
These types of problems require an object to help you keep count. elements are the keys, and values are the number of occurances. Please revisit this problem and pattern as it comes up in a lot problems.